test/OverwhelmingSupportAutoDelegate.t.sol

```diff
@@ -336,3 +336,91 @@ contract OZGovernorBlockNumberModeSetSupportThreshold is OverwhelmingSupportAuto
     );
   }
 }
+
+abstract contract CheckVoteRequirementsTest is OverwhelmingSupportAutoDelegateTest {
+  function testFuzz_CheckVoteRequirements(
```

Comment:
Needs a more expressive name for scopelint

Reply:
Done: 3d7a520307fe8a12df63ece44b2671ceaec41bb5